src/components/skeleton/skeleton-canvas.module.css

```diff
@@ -0,0 +1,6 @@
+.container {
```

Comment:
Are we intentionally moving away from having a separate styles folder? I admit I liked having the folder because it was easier to get an overview of the components in the folder. But if this is the new convention, then I'm fine with it too.